lab-abigail/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-08",
+  "version": "1.0.0",
+  "description": "",
+  "main": "server.js",
+  "directories": {
+    "test": "test"
+  },
+  "scripts": {
+    "test": "mocha",
+    "start": "node server.js",
+    "watch": "nodemon server.js",
+    "debug": "DEBUG=http* nodemon server.js"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "dependencies": {
+    "bluebird": "^3.5.0",
+    "uuid": "^3.0.1"
+  },
+  "devDependencies": {
```

Comment:
You forgot to install ``chai-http`` as a dev dependency, so that broke things when I tried to run your tests. No points lost for this, but be mindful of remembering to do that before submission.